proto/buf/alpha/registry/v1alpha1/repository_commit.proto

```diff
@@ -163,3 +167,17 @@ message DeleteRepositoryDraftCommitRequest {
 }
 
 message DeleteRepositoryDraftCommitResponse {}
+
+message ListBSRHeadHistoryRequest {
+  // The owner of the repository which the repository branch belongs to.
+  string repository_owner = 1;
+  // The name of the repository which the repository branch belongs to.
+  string repository_name = 2;
+  uint32 page_size = 3;
+  string page_token = 4;
+}
```

Comment:
Add `bool reverse`? And declare the default order in which it this returns commits (most recent to older).

Reply:
Yeah, it makes sense to add reverse now, despite the fact that there currently isn't a use for it.

Reply:
Added -- currently it's ordered in reverse by create time, but it will be in reverse order by parentage once we make the data model changes. Just noting this for documentation purposes :)